r-pkg/R/elasticsearch_eda_funs.R

```diff
@@ -120,13 +120,17 @@ get_counts <- function(field
 #'              to data type for all indexed fields.
 #' @importFrom futile.logger flog.fatal flog.info
 #' @importFrom httr GET content stop_for_status
+#' @importFrom data.table := uniqueN
 #' @param es_host A string identifying an Elasticsearch host. This should be of
 #'                the form \code{[transfer_protocol][hostname]:[port]}. For example,
 #'                \code{'http://myindex.thing.com:9200'}.
 #' @param es_indexes A character vector that contains the names of indexes for
 #'                   which to get mappings. Default, is \code{'_all'}, which means
 #'                   get the mapping for all indexes. Names of indexes can be
 #'                   treated as regular expressions.
+#' @param use_alias A boolean flag that controls whether the true Elasticsearch
```

Comment:
Kill off this flag. We don't need it. If people really want to get in the weeds and hit individual indexes instead of aliases, they can specify that in `es_indexes`. The way you've designed this function, it's already flexible to a mixing of aliases and actual index names.

I want to minimize the number of flags we have to maintain. and the associated forks in the code.